src/router/shortUrl.ts

```diff
@@ -0,0 +1,8 @@
+import express from "express";
+import { getAllUrl, getOneUrl, postUrl } from "../controller";
+
+export const router = express.Router();
+
+router.post("/shortUrl", postUrl);
+router.get("/shortUrl/:id", getOneUrl);
+router.get("/shortUrl", getAllUrl);
```

Comment:
These urls does not match the route in the spec

```suggestion
router.post("/urls", postUrl);
router.get("/urls/:id", getOneUrl);
router.get("/urls", getAllUrl);
```